jobs/quickstart/main.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START quick_start]
+
+package main
```

Comment:
Would you mind adding a package comment? It will show up in GoDoc (godoc.org). It should go on the line immediately before `package main`. For example,

```
// Sample quickstart is an example of using the Google Cloud Platform Jobs API.
package main
```

See https://github.com/golang/go/wiki/CodeReviewComments#package-comments.